backend/routes/routes.js

```diff
@@ -0,0 +1,5 @@
+const fetchPDF = require('../services/fetchPDF');
```

Comment:
I'd put routes.js one level above. The only reason I'd have a routes folder would be if you were having different files such as if it was a ecommerce website with many routes that need seperated product routes, customer routes etc. Hopefully that makes sense.